src/io/flutter/logging/FlutterLogFilterPanel.form

```diff
@@ -36,11 +29,15 @@
           <text value="Regex"/>
         </properties>
       </component>
-      <hspacer id="6d179">
+      <component id="b7426" class="com.intellij.ui.SearchTextField" binding="textExpression" custom-create="true">
```

Comment:
Here is the code insert textExpression directly.

Reply:
👍